packages/infrastructure/kube_cert_issuers/main.tf

```diff
@@ -116,6 +116,17 @@ resource "kubernetes_secret" "cloudflare_api_token" {
   }
 }
 
+// Ensure cert manager is alive and well
+resource "null_resource" "wait_cert_manager_rollout" {
+  provisioner "local-exec" {
+    command = join(" && ", [
+      "kubectl rollout status deployment cert-manager-webhook -n cert-manager --timeout=2m",
+      "kubectl rollout status deployment cert-manager-cainjector -n cert-manager --timeout=2m"
+    ])
+  }
+}
```

Comment:
I believe this will break but I like the idea. It may or may not be needed though.